### DISCUSSION TITLE
Remove or change build file names

### DISCUSSION THREAD

**Discussion by sequensucks** [nuxt/nuxt#24610] [Questions]
Need to deploy an app to a several VM and need to have a same build (same file names).
Ideal case is replace a hash with a timestamp.
Try this to remove a hash, but result without changes.
`vite: {
      build: {
        rollupOptions: {
          output: {
            output: {
              entryFileNames: `[name].js`,
              chunkFileNames: `[name].js`,
              assetFileNames: `[name].[ext]`,
            },
          },
        },
      },
    },`
    
    Could someone help me pls

**Comment by xjccc** [comment]
are you add extra `output`?

```
export default defineNuxtConfig({
  vite: {
    build: {
      rollupOptions: {
        output: {
          chunkFileNames: '_nuxt/chunks/[name].js',
          entryFileNames: '_nuxt/js/[name].js',
          assetFileNames: '_nuxt/[name].[ext]'
        }
      }
    }
  }
})
```

**Comment by sequensucks** [comment]
Nice, it works. Thank you!
